### ISSUE TITLE
No module named 'pwd' in Windows

### ISSUE THREAD

**Issue by izkgao** [sdss/marvin#590]
Like [#429](https://github.com/sdss/marvin/issues/429). I followed [the instruction](https://sdss-marvin.readthedocs.io/en/stable/installation.html#marvin-on-windows) trying to import marvin into a jupyter notebook on a Windows 10 machine, but this error occurred. Before importing marvin, I have run these lines of code:

```
import os
os.environ['HOME'] = 'C:\\Users\\Isaac\\Desktop\\py\\marvin'
os.environ['SAS_BASE_DIR'] = os.path.join(os.getenv("HOME"), 'sas')
```
This is the screenshot of the error while importing marvin:

![image](https://user-images.githubusercontent.com/16133788/50622656-db6e9f00-0f49-11e9-9ff6-5425d2f0f1e4.png)

**Desktop:**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 70.0.3538.110

**Comment by havok2063** [comment]
@izkgao thanks for reporting this.  We're aware of the `pwd` issue and have a PR #398 in place to fix this, but it's not quite ready to merge in yet.  We will aim to do this soon.

**Comment by havok2063** [comment]
This has now been resolved with the PR #652 and merged in.  It will be available in the next release of Marvin.
